_sources/Conditionals/Theinandnotinoperators.rst

```diff
@@ -0,0 +1,41 @@
+..  Copyright (C)  Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris
+    Meyers, and Dario Mitchell.  Permission is granted to copy, distribute
+    and/or modify this document under the terms of the GNU Free Documentation
+    License, Version 1.3 or any later version published by the Free Software
+    Foundation; with Invariant Sections being Forward, Prefaces, and
+    Contributor List, no Front-Cover Texts, and no Back-Cover Texts.  A copy of
+    the license is included in the section entitled "GNU Free Documentation
+    License".
+
+.. qnum::
+   :prefix: condition-4-
+   :start: 1
+
+The ``in`` and ``not in`` operators
```

Comment:
I think we need to add the list examples to this section, it keeps the parallelism we've been using with exposing sequence stuff.